libphal/utils_pdbg.C

```diff
@@ -48,5 +74,59 @@ struct pdbg_target *getFsiTarget(struct pdbg_target *proc)
 	return fsi;
 }
 
+/**
+ * @brief Used to return in callback function which are used to get
+ * pdbg taret based on  binary path.
+ *
+ * @param[in] target current device tree target
+ * @param[out] appPrivData used for accessing|storing from|to application
+ *
+ * @return 0 to continue traverse, non-zero to stop traverse
+ **/
+int pdbgCallbackToGetTgt(struct pdbg_target *tgt, void *appPrivData)
```

Comment:
can we make "void *appPrivData"  as "TargetInfo *targetInfo" as I do not see this method being called for different purposes

Reply:
This callback function , and pdbg looking for this prototype.